### ISSUE TITLE
feature news component 

### ISSUE THREAD

**Issue by bdolor** [BCcampus/bcc-sage#151]
## Submit a feature request or bug report

- [x] This is a feature request
- [ ] This is a bug report
- [ ] This request isn't a duplicate of an [existing issue](https://github.com/BCcampus/bcc-sage/issues)
- [ ] You cleared your Web Browsers cache to ensure all new styling and images were loaded. 

## Feature Request

This component displays on two separate landing pages, homepage and news landing page. The functionality is the same but the display is different. The image is a featured image. The date published and an excerpt. 

Possibly a partial.

**Comment by bdolor** [comment]
complete via https://github.com/BCcampus/bcc-sage/commit/754005c1369c62a384c0e569c12da103ab1a2699

can be validated at 
- https://helga.bccampus.ca/wp/bcc/news/
- https://helga.bccampus.ca/wp/bcc/

**Comment by murphybarb** [comment]
Hi @bdolor, I have comments on this one:  

**News Feature on home page: https://helga.bccampus.ca/wp/bcc/**

- The width of the feature container news is wider than in the design mockup. I know we talked about the width of the "Topics of practice" container beside it when it was being validated, but seeing the actual feature news container in place makes me realize how much wider it is than what was intended in the design mockup. This could introduce issues with the images. Is there anything we can do to get closer to the width indicated in the design mockup?


**News Feature on News Landing page: https://helga.bccampus.ca/wp/bcc/news/**

- Design comments, see below. I've attached a screen capture for reference 



**Styling to be addressed in the design review:**

- homepage news feature box – the font for the date is to be regular weight, right now it is light. The font for the headline to remain as is, the light weight.  

- News landing page –  the purple banner containing the title should match the width of the picture. There should be a grey border that encloses the intro text/body copy below the image and banner.


![screen shot 2018-09-07 at 11 27 00 am](https://user-images.githubusercontent.com/11672469/45236586-15c47a00-b291-11e8-9b2a-67593ec0a2bb.png)
